python/mxnet/ndarray/ndarray.py

```diff
@@ -3125,6 +3128,26 @@ def _get_dim_size(start, stop, step):
     return dim_size
 
 
+def _get_slice_len_for(slc, seq_length):
```

Comment:
nit: change `_get_slice_len_for` to `_get_slice_len`.

Reply:
Let me fix it in the other PR (https://github.com/apache/incubator-mxnet/pull/16902)